modules/dists/ReplicatedDist.chpl

```diff
@@ -177,10 +177,10 @@ when the constructor encounters an error.
 
 */
 class ReplicatedDist : BaseDist {
+  var targetLocDom : domain(Locales.idxType);
```

Comment:
Can you say more about what motivated this change to always associative targetLocDom ? I don't see anything about why it was done in the PR message.